src/main/java/seedu/address/logic/commands/DeleteStudentCommand/DeleteStudentCommand.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.logic.commands.deletestudentcommand;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Abstract class for DeleteStudentCommand
+ */
+public abstract class DeleteStudentCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete_student";
```

Comment:
Isit possible to update the COMMAND_WORD to "/delete_student" as planned?

Reply:
Updated. Thanks!